ui/partials/main/thPinboardPanel.html

```diff
@@ -63,26 +63,29 @@
 
 <!-- Save UI -->
 <div id="pinboard-controls" class="btn-group-vertical"
-     title="{{!hasPinnedJobs() ? 'No jobs pinned' : ''}}">
+     title="{{!hasPinnedJobs() ? 'No pinned jobs' : ''}}">
   <div class="btn-group">
     <button class="btn btn-default btn-xs save-btn"
-          title="{{ !hasPinnedJobs() ? 'No job selected' : 'Save all classification data' }}"
-          ng-click="save()"
-          ng-disabled="!canSaveClassifications()">save
+            title="{{ saveUITitle('classification') }}"
+            ng-click="save()"
+            ng-disabled="!user.loggedin || !canSaveClassifications()">save
```

Comment:
Ok, that console error is sorted. I've declared `rootScope.user` earlier, in controllers/main.js. I was going to pull its equivalent in directives/persona.js, but Perf needs it, so I'll leave it there.